lib/postgresql.js

```diff
@@ -72,13 +72,10 @@ function PostgreSQL(postgresql, settings) {
   // this._models = {};
   // this.settings = settings;
   this.constructor.super_.call(this, 'postgresql', settings);
+  this.clientConfig = settings;
   if (settings.url) {
     // pg-pool doesn't handle string config correctly
```

Comment:
Line 76? That is still accurate. Pg-pool expects a different key name than the pg module.